src/pages/settings/expense-categories/components/CreateExpenseCategoryModal.tsx

```diff
@@ -100,7 +100,10 @@ export function CreateExpenseCategoryModal(props: Props) {
       onClose={() => props.setVisible(false)}
       stopPropagationInHeader
     >
-      <div onClick={(event) => event.stopPropagation()}>
+      <div
+        className="flex flex-col space-y-4"
+        onClick={(event) => event.stopPropagation()}
```

Comment:
What are we stopping here? Div shouldn't have any default action attached?

Reply:
@beganovich Yes, we can move this to RouterLink component and avoid having a div here. The purpose of stopping propagation is to prevent unnecessary actions such as opening the slider.